app/javascript/vue/components/ui/VSwitch.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="switch-radio">
+  <div :class="{ 'switch-radio': true, 'wrap-switch': options.length > 4 }">
```

Comment:
I think this is a good idea when switch has many options. Perhaps this could be configurable by prop? Maybe in some scenarios we won't want to break the switch. So, we could add two props `wrap` that will allow wrap tabs and another one to set the max tabs before wrap, like you have now, for example: 0 = always wrap

or you can just use "wrap" for both things, if is passed as prop without assign value, then it will wrap everything, is you set `wrap="4"` then add wrap class when you have 4 or more options